### PR TITLE
In detail views, show main object info in a vertical list instead of a one-row table

### PR DESCRIPTION
**Pull request by jwasinger**
**Resolves issues #446, #704, #332, #828, #831 and #768*.**

*Changed how main object is rendered in detail view to make it not look like a table.  'Info' row is also removed as it doesn't do anything useful. 

\* Examples of using '!' to negate a query still need to be added.

### CODE REVIEW

**Comment by drkitty** [comment]
Info dump:

- In general, we attempt to follow [these commit message guidelines](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
- The "Please review" label I talked about can be accessed by clicking the Labels button directly to the right of the pull request description.
- The manual incrementing of `i` is a little confusing. It's preferable to do any serious data mangling in the view rather than in the template. We don't always follow this rule, but we ought to (and I'm going to fix the existing offenders eventually).

**Comment by drkitty** [comment]
If you're fixing multiple issues in one pull request, note that in the pull request description.

File: cyder/templates/base/tables.html [hunk 1]
```diff
@@ -45,3 +45,48 @@
...
+        {% set header, sort_field = object_table['headers'][loop.index0] %}
+         
+        <tr>
+          <td class="{{ header | lower }}_column"><b>{{ header }}</b></td>
+          
+          {% if col['value'] %}
+            <td>
+            {% for cell in col['value'] %}
+              {% if col['url'][loop.index0] %}
+                <a class="info"
```
Comment (church29) [inline cyder/templates/base/tables.html]: "<a class="info"" shouldn't be there, unless it is specifically for the info column.

**Comment by church29** [comment]
System detail is still displayed in the old format. Also, there needs to be spacing above and below the new table. The title for the following table seems to be just another entry.

**Comment by jwasinger** [comment]
With regards to the system detail view, it appears that there could be more than one static/dynamic interface pertaining to a system.  However, I cannot find any instances of this being the case.  Should I leave the format for static/dynamic interface tables the same?

**Comment by jwasinger** [comment]
Also in system detail view:  If I am removing the Info/Actions fields from the main object, that leaves only the Name field and the name is already displayed above the main object table.  I am going to remove the table entirely.

**Comment by church29** [comment]
Definitely leave the static/dynamic interface tables as is. We switched to the system/interface model for the case of having multiple entries per system. I think removing the table entirely is acceptable. The only entry for system is name correct?

**Comment by jwasinger** [comment]
Yes.  And name is already displayed at the top of the page

**Comment by church29** [comment]
Looks great. Well done.

**Comment by drkitty** [comment]
If you think this is ready for someone to review and merge, add the "please review" tag.

**Comment by akeym** [comment]
WRT #704 - I still see a circle appear really quickly when I type in the create attribute form (has an autocomplete box)

**Comment by akeym** [comment]
WAIT - wrong. i'm wrong.  I was in the wrong tab when testing... sigh. its fine.

**Comment by drkitty** [comment]
Needs a little more horizontal padding between the field name column and the data column.

**Comment by drkitty** [comment]
Also needs a bit more vertical padding below the main object info table.

**Comment by drkitty** [comment]
Maybe something like `border-spacing: 16px 0`.

**Comment by drkitty** [comment]
Otherwise, looks good.

**Comment by murrown** [comment]
Can we add `Container` to the fields shown in all detail views?

**Comment by drkitty** [comment]
We'd do that in each model's `details()` method, right?

**Comment by drkitty** [comment]
Everything appears to work, so I'm going to merge this. We can fix the cosmetic problems and display the "container" field later.

**Pull request merged**